addons/mail/static/src/core/web/messaging_menu.js

```diff
@@ -120,64 +120,12 @@ export class MessagingMenu extends Component {
         };
     }
 
+    get threads() {
```

Comment:
is it needed to have this getter? `this.threads` seems not making a lot of differences from calling `this.getThreads();` directly

Reply:
Change in stable version, so I keep the getter as it was existing in template (see onWillRender negative diff)